json/src/transaction.rs

```diff
@@ -46,7 +46,7 @@ pub struct Transaction {
 	pub v: MaybeEmpty<Uint>,
 	/// Secret
 	#[serde(rename = "secretKey")]
-	pub secret: Option<H256>,
+	pub secret: H256,
```

Comment:
not sure about this breaking change

Reply:
You mean before you could call the RPC without this param and after this you can't?